src/com/amazon/corretto/crypto/provider/AmazonCorrettoCryptoProvider.java

```diff
@@ -491,12 +491,9 @@ public void assertHealthy() throws RuntimeCryptoException {
     selfTestSuite.assertAllTestsPassed();
   }
 
-  /**
-   * Returns {@code true} if and only if the underlying libcrypto library is a FIPS build and passes
-   * FIPS KAT self tests
-   */
+  /** Returns {@code true} if and only if the underlying libcrypto library is a FIPS build */
   public boolean isFips() {
```

Comment:
I'm reverting this to its old behavior as in 2.0.0; if a customer need to run the KAT self tests, they can invoke `runSelfTests` and it would run all self tests including KAT ones.